### ISSUE TITLE
internal error installing weakauras-companion

### ISSUE THREAD

**Issue by hauk92** [layday/instawow#46]
I tried installing the WeakAuras aura updater via
```
instawow weakauras-companion build -a <your account name> (account name is correct)
instawow install instawow:weakauras-companion
```
and get the following error.

Output:
```
C:\Users\Blabla>instawow install instawow:weakauras-companion
! instawow:weakauras-companion
  internal error
```

error.log:
```
2020-09-08 12:55:01.902 | ERROR    | instawow.manager:_capture_exc:265 - unclassed error
Traceback (most recent call last):

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
           |         |     -> {'__name__': '__main__', '__doc__': None, '__package__': '', '__loader__': <zipimporter object "C:\Users\Blabla\.local\bin\ins...
           |         -> <code object <module> at 0x0000026DB3F74DF0, file "C:\Users\Blabla\.local\bin\instawow.exe\__main__.py", line 2>
           -> <function _run_code at 0x0000026DB3D78B80>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
         |     -> {'__name__': '__main__', '__doc__': None, '__package__': '', '__loader__': <zipimporter object "C:\Users\Blabla\.local\bin\ins...
         -> <code object <module> at 0x0000026DB3F74DF0, file "C:\Users\Blabla\.local\bin\instawow.exe\__main__.py", line 2>

  File "C:\Users\Blabla\.local\bin\instawow.exe\__main__.py", line 7, in <module>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\click\core.py", line 829, in __call__
    return self.main(*args, **kwargs)
           |    |     |       -> {}
           |    |     -> ()
           |    -> <function BaseCommand.main at 0x0000026DB5C56CA0>
           -> <Group main>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\click\core.py", line 782, in main
    rv = self.invoke(ctx)
         |    |      -> <click.core.Context object at 0x0000026DB66409D0>
         |    -> <function MultiCommand.invoke at 0x0000026DB5C57A60>
         -> <Group main>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\click\core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
           |               |       |       |      -> <click.core.Context object at 0x0000026DB40C19A0>
           |               |       |       -> <function Command.invoke at 0x0000026DB5C57670>
           |               |       -> <Command install>
           |               -> <click.core.Context object at 0x0000026DB40C19A0>
           -> <function MultiCommand.invoke.<locals>._process_result at 0x0000026DB3B461F0>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\click\core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           |   |      |    |           |   -> {'addons': [Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, str...
           |   |      |    |           -> <click.core.Context object at 0x0000026DB40C19A0>
           |   |      |    -> <function install at 0x0000026DB66BE4C0>
           |   |      -> <Command install>
           |   -> <function Context.invoke at 0x0000026DB5C56790>
           -> <click.core.Context object at 0x0000026DB40C19A0>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\click\core.py", line 610, in invoke
    return callback(*args, **kwargs)
           |         |       -> {'addons': [Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, str...
           |         -> ()
           -> <function install at 0x0000026DB66BE4C0>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\click\decorators.py", line 33, in new_func
    return f(get_current_context().obj, *args, **kwargs)
           | |                           |       -> {'addons': [Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, str...
           | |                           -> ()
           | -> <function get_current_context at 0x0000026DB59D8D30>
           -> <function install at 0x0000026DB66BE430>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\cli.py", line 288, in install
    results = obj.m.run(obj.m.install(addons, replace))
              |   | |   |   | |       |       -> False
              |   | |   |   | |       -> [Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, strategy_vals=...
              |   | |   |   | -> <function Manager._with_lock.<locals>.outer.<locals>.inner at 0x0000026DB6DEE5E0>
              |   | |   |   -> <instawow.manager.CliManager object at 0x0000026DB6E2F160>
              |   | |   -> <instawow.cli.ManagerWrapper object at 0x0000026DB40D2640>
              |   | -> <function CliManager.run at 0x0000026DB6DEEB80>
              |   -> <instawow.manager.CliManager object at 0x0000026DB6E2F160>
              -> <instawow.cli.ManagerWrapper object at 0x0000026DB40D2640>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 772, in run
    return loop.run_until_complete(run())
           |    |                  -> <function CliManager.run.<locals>.run at 0x0000026DB7C51AF0>
           |    -> <function BaseEventLoop.run_until_complete at 0x0000026DB5E67EE0>
           -> <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\asyncio\base_events.py", line 603, in run_until_complete
    self.run_forever()
    |    -> <function BaseEventLoop.run_forever at 0x0000026DB5E67E50>
    -> <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\asyncio\base_events.py", line 570, in run_forever
    self._run_once()
    |    -> <function BaseEventLoop._run_once at 0x0000026DB5E6A9D0>
    -> <_WindowsSelectorEventLoop running=True closed=False debug=False>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\asyncio\base_events.py", line 1859, in _run_once
    handle._run()
    |      -> <function Handle._run at 0x0000026DB5E09700>
    -> <Handle <TaskWakeupMethWrapper object at 0x0000026DB7C43A60>(<_GatheringFu...d argument')]>)>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\asyncio\events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
    |    |            |    |           |    -> <member '_args' of 'Handle' objects>
    |    |            |    |           -> <Handle <TaskWakeupMethWrapper object at 0x0000026DB7C43A60>(<_GatheringFu...d argument')]>)>
    |    |            |    -> <member '_callback' of 'Handle' objects>
    |    |            -> <Handle <TaskWakeupMethWrapper object at 0x0000026DB7C43A60>(<_GatheringFu...d argument')]>)>
    |    -> <member '_context' of 'Handle' objects>
    -> <Handle <TaskWakeupMethWrapper object at 0x0000026DB7C43A60>(<_GatheringFu...d argument')]>)>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 768, in run
    return await awaitable
                 -> <coroutine object Manager._with_lock.<locals>.outer.<locals>.inner at 0x0000026DB6E106C0>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 336, in inner
    return await coro_fn(self, *args, **kwargs)
                 |       |      |       -> {}
                 |       |      -> ([Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, strategy_vals...
                 |       -> <instawow.manager.CliManager object at 0x0000026DB6E2F160>
                 -> <function Manager.install at 0x0000026DB6DEE550>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 606, in install
    results = {d: await _capture_exc(c) for d, c in result_coros.items()}
                        |                           |            -> <method 'items' of 'dict' objects>
                        |                           -> {Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, strategy_vals=...
                        -> <function _capture_exc at 0x0000026DB6DEC940>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 606, in <dictcomp>
    results = {d: await _capture_exc(c) for d, c in result_coros.items()}
               |        |            |      |  -> <function _error_out.<locals>.inner at 0x0000026DB93B63A0>
               |        |            |      -> Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, strategy_vals=())
               |        |            -> <function _error_out.<locals>.inner at 0x0000026DB93B63A0>
               |        -> <function _capture_exc at 0x0000026DB6DEC940>
               -> Defn(source='instawow', source_id=None, name='weakauras-companion', strategy=<Strategies.default: 'default'>, strategy_vals=())

> File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 258, in _capture_exc
    return await coro()
                 -> <function _error_out.<locals>.inner at 0x0000026DB93B63A0>

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 249, in inner
    raise error
          -> OSError(22, 'Invalid argument')

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\manager.py", line 131, in download_archive
    await copy_async(unquote(url[7:]), dest)
          |          |       |         -> WindowsPath('C:/Users/Blabla/AppData/Local/Temp/instawow/cache/f709d0aa5646dca7f57904e1be899b61')
          |          |       -> 'file:///C:/Users/Blabla/AppData/Roaming/instawow/profiles/__default__/plugins/instawow.wa_updater/WeakAurasCompanion.zip'
          |          -> <function unquote at 0x0000026DB41448B0>
          -> <function copy at 0x0000026DB6DEC040>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\concurrent\futures\thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
             |        |            -> None
             |        -> None
             -> None

  File "c:\users\blabla\.local\pipx\venvs\instawow\lib\site-packages\instawow\utils.py", line 173, in <lambda>
    return loop.run_in_executor(None, lambda: fn(*args, **kwargs))
                                              |   |       -> {}
                                              |   -> ('/C:/Users/Blabla/AppData/Roaming/instawow/profiles/__default__/plugins/instawow.wa_updater/WeakAurasCompanion.zip', WindowsP...
                                              -> <function copy at 0x0000026DB5C6BA60>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\shutil.py", line 415, in copy
    copyfile(src, dst, follow_symlinks=follow_symlinks)
    |        |    |                    -> True
    |        |    -> WindowsPath('C:/Users/Blabla/AppData/Local/Temp/instawow/cache/f709d0aa5646dca7f57904e1be899b61')
    |        -> '/C:/Users/Blabla/AppData/Roaming/instawow/profiles/__default__/plugins/instawow.wa_updater/WeakAurasCompanion.zip'
    -> <function copyfile at 0x0000026DB5C6B820>

  File "c:\users\blabla\appdata\local\programs\python\python38\lib\shutil.py", line 261, in copyfile
    with open(src, 'rb') as fsrc, open(dst, 'wb') as fdst:
              |                        -> WindowsPath('C:/Users/Blabla/AppData/Local/Temp/instawow/cache/f709d0aa5646dca7f57904e1be899b61')
              -> '/C:/Users/Blabla/AppData/Roaming/instawow/profiles/__default__/plugins/instawow.wa_updater/WeakAurasCompanion.zip'

OSError: [Errno 22] Invalid argument: '/C:/Users/Blabla/AppData/Roaming/instawow/profiles/__default__/plugins/instawow.wa_updater/WeakAurasCompanion.zip'
```

**Comment by layday** [comment]
Thank you for the report.  Windows paths were not handled correctly.

**Comment by layday** [comment]
Fixed in v1.14.1.  Thanks again.
